packages/lit-html/src/index.js

```diff
@@ -416,6 +417,28 @@ function isIterable(iterator) {
   );
 }
 
+/**
+ * Print an error message that shows what HTML element got
+ * unexpected primitive rendered to part, and thus failed to hydrate.
+ * @param { unknown } value
+ * @param { Comment } marker
+ * @param { RenderOptions } [options]
+ */
+function consoleUnexpectedPrimitiveError(value, marker, options) {
+  try {
```

Comment:
Do we need the try / catch?

Reply:
Yes, or IMO we should, as I unfortunately don't know this lib well enough, and it is not written in TS, so I am afraid e.g. `marker` on new line 433 for example in some cases not has a `parentElement` prop, and I probably should get to know the details entirely before suggesting this, but I would just not enjoy me causing some unhandled error only due to a log message we want to tailor 🤓  🙈